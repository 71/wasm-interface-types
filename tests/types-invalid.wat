;; parse-fail

(module
  (@interface type foo))

(; CHECK-ALL:
expected `(`
     --> tests/types-invalid.wat:4:20
      |
    4 |   (@interface type foo))
      |                    ^
;)
