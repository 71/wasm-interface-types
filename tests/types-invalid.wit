;; parse-fail

(module
  (@interface type foo))
