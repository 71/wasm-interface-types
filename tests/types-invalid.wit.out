expected `(`
     --> ../../tests/types-invalid.wit:4:20
      |
    4 |   (@interface type foo))
      |                    ^
